constantine-rust/constantine-halo2-zal/Cargo.toml

```diff
@@ -10,7 +10,8 @@ repository = "https://github.com/mratsim/constantine"
 [dependencies]
 constantine-sys = { path = "../constantine-sys" }
 constantine-core = { path = "../constantine-core" }
-halo2curves = { git = 'https://github.com/taikoxyz/halo2curves', branch = "pr-pse-exec-engine" }
+halo2_middleware = { git = 'https://github.com/privacy-scaling-explorations/halo2' }
+halo2curves = { version = "0.6.1", default-features = false }
```

Comment:
Do we want to depend explicitly on `0.6.1`?

Reply:
Actually the way cargo works, this means a dependency on 0.6.x: https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html

![image](https://github.com/mratsim/constantine/assets/22738317/ead10ffa-f5a7-4af9-a7fe-94f791883708)

But I think the elliptic curves and CurveAffine should be reexported by Zal or Middleware.

Reply:
Ah, that clears up a lot, thanks!